packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -684,6 +728,24 @@ export abstract class BaseService extends Resource
     };
   }
 
+  /**
+   *   Add an alarm based on ECS metric
+  */
+  public createEcsMetricAlarm(metric: EcsMetric, ecsMetricAlarmProps?: EcsMetricAlarmProps): cloudwatch.Alarm {
+    // Throw an error if service connect is not configured
+    if (!this._serviceConnectConfig) {
+      throw new Error('Service connect must be enabled to set service connect metric alarms.');
+    }
+    const ecsMetric = this.metric(metric, ecsMetricAlarmProps?.metricProps);
+    const metricAlarm = new cloudwatch.Alarm(this, `${metric}Alarm`, ecsMetricAlarmProps?.useAsDeploymentAlarm && ecsMetricAlarmProps?.alarmProps ? ecsMetricAlarmProps.alarmProps : {
+      metric: ecsMetric as IMetric,
+      threshold: ecsMetricAlarmProps?.alarmProps?.threshold || 1,
+      evaluationPeriods: ecsMetricAlarmProps?.alarmProps?.evaluationPeriods || 1,
```

Comment:
We should probably provide better defaults for these. 
```suggestion
      threshold: ecsMetricAlarmProps?.alarmProps?.threshold || 85,
      evaluationPeriods: ecsMetricAlarmProps?.alarmProps?.evaluationPeriods || 3,
```

We should also be sure to provide good defaults for the metric options. 

https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-cloudwatch.Metric.html
https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-cloudwatch.AlarmProps.html#metric

Reply:
Gotcha setting threshold value to  85 and evaluationPeriods to 3
so that  every deployment will not fail because CPU utilization percent is going to be greater than 1%.